698.add confirmation feedback form.youssef mlika

CrypticCoder67458 opened the pull request.

chgad commented:
Closes #698

chgad commented on src/locales/de.json:
```diff
@@ -253,7 +253,8 @@
       "question": "Frage"
-    }
+    },
+    "gdpr": "Ich stimme zu, dass Administratoren gemäß unseren internen Regeln auf meine Daten in der Datenbank zugreifen dürfen."
```
@sheepyhollow check woording pls.

chgad commented on src/locales/en.json:
```diff
@@ -253,7 +253,8 @@
       "question": "Question"
-    }
+    },
+    "gdpr": "I agree to allow administrators to access my data in the database, as required by our internal rules."
```
@sheepyhollow see above

chgad reviewed (commented):
After revision of woording lgtm

chgad reviewed (changes requested):
There is the followoing problem with this implementation:
1. Open the form in developer tool with touch simmulation on
2. Click on start-date selection
3. Select date, calendar widget closes
4. Click on it again
---> The calendar widget briefly opens and closes again
5. Click on it again 
---> The calendar widget opens as usual

CrypticCoder67458 commented:
ok i will look into the issue

CrypticCoder67458 commented:
i fixed the issue 

> There is the followoing problem with this implementation:
> 
>     1. Open the form in developer tool with touch simmulation on
> 
>     2. Click on start-date selection
> 
>     3. Select date, calendar widget closes
> 
>     4. Click on it again
>        ---> The calendar widget briefly opens and closes again
> 
>     5. Click on it again
>        ---> The calendar widget opens as usual

i fixed the issue and created a new pull request can you check it.

chgad commented:
The last 4 Commits are simply wrong, i will remove them before merging.
@CrypticCoder67458 I think I need to explain Caddy/Environment a bit more indepth.
